package.json

```diff
@@ -18,9 +18,10 @@
 		"@rollup/plugin-json": "6.1.0",
 		"@rollup/plugin-node-resolve": "16.0.0",
 		"@types/vscode": "1.80.0",
+		"@vscode/test-electron": "^2.4.1",
 		"@vscode/vsce": "3.2.1",
 		"lefthook": "1.10.8",
-		"ovsx": "0.10.1",
+		"rimraf": "^6.0.1",
```

Comment:
As far as I can tell, the output always gets overwritten. What's the value in adding this

Reply:
> As far as I can tell, the output always gets overwritten. What's the value in adding this

If it automatically deletes the contents of the out folder then there is no point in doing this

Reply:
Done - [`8c5a59d` (#478)](https://github.com/biomejs/biome-vscode/pull/478/commits/8c5a59de08ce6c73bfa4b0c5a3faf3c86abb3af5)

Reply:
> the output always gets overwritten

No, you're wrong, I used the `rollup-plugin-delete` plugin for that